app/client/components/helpCentre/helpCentrePageTitle.tsx

```diff
@@ -0,0 +1,23 @@
+import { Location } from "@reach/router";
+import React from "react";
+
+interface PageTitleProps {
+  title?: string | undefined;
+}
+
+export const PageTitle = (props: PageTitleProps) => (
```

Comment:
The standard for this project is that the main component you are exporting is reflected in the file name. So in this case the file name would be `pageTitle.tsx`